manager/manifests/statsd.yaml

```diff
@@ -55,13 +55,18 @@ metadata:
   name: cloudwatch-agent-statsd
   namespace: default
 spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
   selector:
     matchLabels:
       name: cloudwatch-agent-statsd
   template:
     metadata:
       labels:
         name: cloudwatch-agent-statsd
+        hallo: dummy
```

Comment:
This is probably left over from testing :)

Reply:
will correct